packages/ubus-lime-utils/tests/test_hotspot_wwan.lua

```diff
@@ -71,6 +88,67 @@ describe('hotspot_wwan tests #hotspot_wwan', function()
         assert.is.equal(-66, status.signal)
     end)
 
+    it('test is_safe is false when no mesh ifaces configured', function()
+        uci:set('wireless', 'radio0', 'wifi-device')
+        uci:set('wireless', 'radio0', 'type', 'mac80211')
+        uci:set('wireless', 'radio0', 'channel', '4')
+        uci:set('wireless', 'radio0', 'hwmode', '11a')
+        uci:set('wireless', 'radio0', 'macaddr', '01:23:45:67:89:AB')
+        uci:set('wireless', 'radio0', 'disabled', '0')
+        uci:commit('wireless')
+
+        local is_safe = hotspot_wwan._is_safe('internet', 'psk2', 'radio0')
+        assert.is_true(is_safe)
+    end)
+
+    it('test is_safe is false when mesh ifaces configured', function()
```

Comment:
It doesn't seem to me that we are setting up a "mesh iface" over radio 0 in this test

Reply:
right.... did a mistake when refactored the code to remove duplication in the tests

Reply:
fixed